src/sources/file.rs

```diff
@@ -112,12 +113,14 @@ pub struct FileConfig {
 
     /// Ignore files with a data modification date older than the specified number of seconds.
     #[serde(alias = "ignore_older", default)]
+    #[configurable(metadata(docs::examples = "example_seconds()"))]
     pub ignore_older_secs: Option<u64>,
 
     /// The maximum number of bytes a line can contain before being discarded.
```

Comment:
Max size of a line before it will be discarded?

I still struggle a bit knowing when to remove the units from the doc comment.

Reply:
Resolved in [192c429](https://github.com/vectordotdev/vector/pull/16099/commits/192c429cac97c0da46a361108a4d292eeef97f2a)